_scripts/migrations/0014_add_post_created_at_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS posts_created_at_askenkadss12vv ON posts (discussion_id, created_at);
```

Comment:
Adds an index to the posts table to optimize the retrieval by created_at. Can also be used to optimize uniqueness verification for the couple (discussion_id, created_at) in order to avoid paging inconsistencies in which two or more posts are created in the same exact time. The problem will be targeted in the future, if necessary.